src/main.rs

```diff
@@ -42,8 +42,8 @@ fn main() -> Result<(), JaktError> {
         }
     }
 
-    if let Some(error) = first_error {
-        Err(error)
+    if first_error.is_some() {
+        std::process::exit(1)
```

Comment:
Just curious, what's the reason for this?

Reply:
Ahh, this is a cleanup of what we had before. Previous, we panicked if there was an error, which prints debug info to the screen (after the wire)

I believe the more correct solution is non-zero exit code